packages/vue-server-renderer/types/index.d.ts

```diff
@@ -0,0 +1,44 @@
+import Vue = require('vue');
+import { Readable } from 'stream';
+
+export declare function createRenderer(options?: RendererOptions): Renderer;
+
+export declare function createBundleRenderer(bundle: any, options?: BundleRendererOptions): BundleRenderer;
+
+type RenderCallback = (err: Error, html: string) => void;
```

Comment:
`err` is nullable, if user turns on `strictNullChecks` TS will wrongly report `if (err === null) {}` as unreachable error.
(edit: strangely it does not report in 2.3.3)

https://github.com/vuejs/vue/blob/d333a498e93acf2d2844fd396e8544658958a3c6/src/server/create-renderer.js#L74

While `html` is also nullable, I think it is more desirable to declare it as `string` since users should use it after checking `err`. Declaring `html` as nullable will force users to unnecessarily recheck because TS cannot relate two variables in flow control :(